CHANGELOG.md

```diff
@@ -3,6 +3,106 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v7.0.0-beta.2
+
+_Feb 9, 2024_
+
+We'd like to offer a big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 🏃 Add slot typings on the Data Grid components (#11795) @romgrk
+- 🎁 Support UTC date formatting in Charts tooltip (#11943) @shaharyar-shamshi
+- 🌍 Improve Danish (da-DK) locale Data Grid (#11877) @ShahrazH
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+### Breaking changes
+
+- If you're using the [commercial license](https://next.mui.com/x/introduction/licensing), you need to update the import path:
```

Comment:
IMHO, this is a pretty good location to mention this BC. 🤔 
It is relevant for both Data Grid and Pickers packages and this way it could be noticed better and sooner.
WDYT @flaviendelangle?

Reply:
IMHO we have 3 solutions here:

1. We apply the same structure for `@mui/x-license` as we do for others, with an `H3` and we put the breaking changes in it, as well as any PR that impacts this package (instead of using `[core]`)

2. We keep the PRs in `[core]` and we put the breaking changes before any package (that's the solution currently applies in this PR)

3. We duplicate the PR and the breaking changes in every set of components with a commercial license (`@mui/x-data-grid-pro` and `@mui/x-date-pickers-pro` as of today)

I would tend to prefer `1` because it also solves correctly the PRs that are non breaking but that are worth noting for users. Right now those PRs are lost in `[core]` which is just like if they were not in the `CHANGELOG` because this section is the thing we tell people not to care about.
But honestly I don't think it's the decision of the century either :laughing:

Reply:
I'm fine with either of 3 options above

Reply:
went with option 1.